### ISSUE TITLE
怎么在输入法engine中获取当前的光标位置信息？

### ISSUE THREAD

**Issue by hantengc** [scim-im/scim#42]
我想要在我的引擎中显示我自己的输入法候选词窗口位置，这个时候我就需要获取一下光标位置信息。
请问scim中有获取当前输入光标基于屏幕的位置信息API吗？

**Comment by hantengc** [comment]
@tzhuan

**Comment by tzhuan** [comment]
No. The scim doesn't export this info to the input method modules currently.

**Comment by hantengc** [comment]
@tzhuan   我怎么样才可以与您取得联系？我的邮箱是1341910529@qq.com
最近打算向scim提一些pr和咨询一些问题  是否可以与您有较好的沟通方式

**Comment by tzhuan** [comment]
As far as I know scim doesn't provide the API for querying the cursor locations. The concept of "cursor location" varies for different toolkits (X, wayland, gtk+, Qt, etc.) and it may not relevant for some toolkit.

Please just open the PR so we can discuss on that PR. Anyone else with interesting may also join the discussion.
